### ISSUE TITLE
registry.terraform.io missing several releases

### ISSUE THREAD

**Issue by jyoungs** [terraform-google-modules/terraform-google-sql-db#680]
### TL;DR

Essentially the same as #650 - 

[Terraform registry](https://registry.terraform.io/modules/terraform-google-modules/sql-db/google/latest) only has 23.0.0

Wonder what I'm doing differently where this isn't more of a problem for others. Would it be better to pull the module directly from GH?

### Expected behavior

All [module releases](https://github.com/terraform-google-modules/terraform-google-sql-db/releases) are available in the registry (specifically 25.0 is referenced in the docs: https://registry.terraform.io/modules/googlecloudplatform/sql-db/google/latest/submodules/postgresql

### Observed behavior

```
➜  cloud-sql-postgresql git:(provider_update) ✗ tf init
Initializing the backend...
Initializing modules...
╷
│ Error: Unresolvable module version constraint
│
│   on main.tf line 60:
│   60: module "postgres" {
│
│ There is no available version of module "registry.terraform.io/terraform-google-modules/sql-db/google//modules/postgresql" (main.tf:60) which matches the given version
│ constraint. The newest available version is 23.0.0.
╵
```

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
Terraform v1.10.2
on darwin_arm64
```
```


### Additional information

_No response_

**Comment by imrannayer** [comment]
synced
